app/layout/help/coming-soon/component.tsx

```diff
@@ -18,12 +18,13 @@ const THEME = {
 };
 export interface ComingSoonProps {
   children: ReactChild;
-  placement?: string;
+  placement?: any;
```

Comment:
Unknown and string type doesn't works, seems to be something inherited from the Tooltip component carried by children.